packages/common/src/store/buy-usdc/sagas.ts

```diff
@@ -170,49 +168,32 @@ function* transferStep({
   usePaymentRouter?: boolean
   memo: string
 }) {
-  const audiusBackendInstance = yield* getContext('audiusBackendInstance')
-  const feePayer = yield* select(getFeePayer)
-  if (!feePayer) {
-    throw new Error('Missing feePayer unexpectedly')
-  }
-  const feePayerOverride = new PublicKey(feePayer)
   const sdk = yield* getSDK()
-  const recentBlockhash = yield* call(getRecentBlockhash, { sdk })
+  const { USDC_MINT_ADDRESS } = yield* getContext('env')
+  const mintPublicKey = new PublicKey(USDC_MINT_ADDRESS)
+  const mintDecimals = USDC(0).decimalPlaces
 
   yield* call(
     retry,
     async () => {
+      console.debug(`Starting transfer transaction...`)
       const transferTransaction = await createTransferToUserBankTransaction(
-        audiusBackendInstance,
+        sdk,
         {
           wallet,
           userBank,
-          mint: 'USDC',
+          mintPublicKey,
+          mintDecimals,
           amount,
-          memo,
-          feePayer: feePayerOverride,
-          recentBlockhash
+          memo
         }
       )
-      transferTransaction.partialSign(wallet)
-
-      console.debug(`Starting transfer transaction...`)
-      const { res, error } = await relayTransaction(audiusBackendInstance, {
-        transaction: transferTransaction
-      })
-
-      if (res) {
-        console.debug(`Transfer transaction succeeded: ${res}`)
-        return
-      }
-
-      console.debug(
-        `Transfer transaction stringified: ${JSON.stringify(
-          transferTransaction
-        )}`
+      transferTransaction.sign([wallet])
+      const res = await sdk.services.solanaClient.sendTransaction(
+        transferTransaction
       )
-      // Throw to retry
```

Comment:
I am assuming we no longer need to explicitly throw because solanaClient is throwing and/or doing retries for us?

Reply:
Yeah tbh don't think we need retries generally but we can circle back on that. But yes this does throw on error